README.md

```diff
@@ -129,6 +129,19 @@ For statistics endpoint:
 curl -w '%{http_code}' http://localhost:8080/ocr-api/statistics
 ```
 
+## Using CHS docker
+
+``` bash
+curl --noproxy '*'  http://api.chs.local/ocr-api/healthcheck
+
+curl --noproxy '*'  http://api.chs.local/ocr-api/statistics
+
+curl --noproxy '*' -w '%{http_code}' --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"app_id": "curl-test","image_endpoint": "http://testurl.com/cff/servlet/viewArticles?transaction_id=9613245852", "converted_text_endpoint": "http://testurl.com/ocr-results/", "response_id": "9613245852"}' \
+  http://chs.local/api.chs.local/api/ocr/image/tiff/extractTextRequest
```

Comment:
I think the url should be http://api.chs.local/api/ocr/image/tiff/extractTextRequest, not http://chs.local/api.chs.local/api/ocr/image/tiff/extractTextRequest